src/main/scala/com/ironcorelabs/GroupGetResult.scala

```diff
@@ -0,0 +1,42 @@
+package com.ironcorelabs.scala.sdk
+
+import com.ironcorelabs.{sdk => jsdk}
+import java.{util => ju}
+
+/**
+ * Group information
+ */
+case class GroupGetResult(
+  id: GroupId,
+  name: Option[GroupName],
+  groupMasterPublicKey: PublicKey,
+  isAdmin: Boolean,
+  isMember: Boolean,
+  adminList: Option[List[UserId]],
```

Comment:
We should add a comment about the difference between None and Some(Nil).

Reply:
Added comments to all parameters